MonsterModifiers/MonsterModifiers.csproj

```diff
@@ -1,253 +1,242 @@
+    <Compile Include="Src\Modifiers\IgnoreArmor.cs" />
+    <Compile Include="Src\Modifiers\Quiet.cs" />
+    <Compile Include="Src\Modifiers\StaggerDeath.cs" />
+    <Compile Include="Src\Modifiers\StaggerImmune.cs" />
+    <Compile Include="Src\Modifiers\DamageModifiers.cs" />
+    <Compile Include="Src\Modifiers\RemoveStatusEffect.cs" />
+    <Compile Include="Src\Modifiers\ShieldBreaker.cs" />
+    <Compile Include="Src\Modifiers\ShieldDome.cs" />
+    <Compile Include="Src\Modifiers\PersonalShield.cs" />
+    <Compile Include="Src\Modifiers\DeathSpawns.cs" />
+    <Compile Include="Src\Modifiers\SoulEater.cs" />
+    <Compile Include="Src\Modifiers\StaminaSiphon.cs" />
+    <Compile Include="Src\Modifiers\Vampiric.cs" />
+    <Compile Include="Src\Modifiers\Wet.cs" />
+    <Compile Include="Src\Patches\AddMonsterModifiersToCharacter.cs" />
+    <Compile Include="Src\Patches\AddVisualsPatch.cs" />
+    <Compile Include="Src\Patches\EnemyHudPatch.cs" />
+    <Compile Include="Src\Plugin.cs" />
+    <Compile Include="Src\StatusEffects\BloodLoss_SE.cs" />
+    <Compile Include="Src\StatusEffects\HealDeath_SE.cs" />
+    <Compile Include="Src\Utils\CompatibilityUtils.cs" />
+    <Compile Include="Src\Utils\DamageUtils.cs" />
+    <Compile Include="Src\Utils\ModifierAssetUtils.cs" />
+    <Compile Include="Src\Utils\ModifierUtils.cs" />
+    <Compile Include="Src\Utils\PrefabUtils.cs" />
+    <Compile Include="Src\Utils\SpawnCommands.cs" />
+    <Compile Include="Src\Utils\StatusEffectUtils.cs" />
+    <Compile Include="Src\Utils\TranslationUtils.cs" />
+    <Compile Include="Src\Utils\WorldUtils.cs" />
+    <Compile Include="Src\Utils\YamlUtils.cs" />
+    <Compile Include="Src\Utils\ZnetUtils.cs" />
+    <Compile Include="Src\Visuals\BowVisuals.cs" />
+    <Compile Include="VersionHandshake.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="App.config" />
+  </ItemGroup>
+  <ItemGroup>
+    <Content Include=".gitignore" />
+    <Content Include="environment.props" />
+    <Content Include="icons\README.txt" />
+    <Content Include="ILRepack.targets" />
+    <Content Include="Libs\LocalizationManager.dll" />
+    <Content Include="LICENSE.md" />
+    <Content Include="README.md" />
+    <Content Include="Thunderstore\CHANGELOG.md" />
+    <Content Include="Thunderstore\icon.png" />
+    <Content Include="Thunderstore\manifest.json" />
+    <Content Include="UpdateManifest.ps1" />
+    <EmbeddedResource Include="assets\modifiericons" />
+    <EmbeddedResource Include="assets\monster_modifiers_ashlands" />
+    <EmbeddedResource Include="assets\statusicon" />
+    <EmbeddedResource Include="Src\Config\modifierValues.yml" />
+    <EmbeddedResource Include="translations\English.yml" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="ILRepack.Lib.MSBuild.Task" Version="2.0.43">
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
```

Comment:
*wdh* I think this was added automatically when updating to the latest version of `ILRepack`. Checking.

Reply:
OK just verified it's fine, since nothing is being repacked (anymore). `YamlDotNet` is now on Thunderstore as a package.